packages/contentstack-import/src/utils/content-type-helper.ts

```diff
@@ -138,7 +147,7 @@ export const removeReferenceFields = async function (
     ) {
       flag.supressed = true;
       schema[i].reference_to = ['sys_assets'];
-      return true;
+      return true; // undo remove
```

Comment:
@antonyagustine Removing line 150 is not part of Raj's solution (he had removed it on my suggestion). Adding `return true` again for now. This will be handled later, while addressing issue 40844

Reply:
@cs-raj and @abhinav-from-contentstack Can we have the above changes also in the same PR if it's just removing return statements only.?

Reply:
> @cs-raj and @abhinav-from-contentstack Can we have the above changes also in the same PR if it's just removing return statements only.?

@antonyagustine , @cs-raj  and @abhinav-from-contentstack  Yes, I think that return is unnecessary here, and by removing it, the prime demo stack issue also resolved.

Reply:
Sure @antonyagustine

Reply:
I've added those changes, please have a look @antonyagustine, @cs-raj